README.md

```diff
@@ -8,5 +8,8 @@ Documentation:
 * [Methods](./BootstrapReport.lyx)
 * [Examples](./examples/)
 
+WebApp:
```

Comment:
> @jmshapir I think maybe you already deployed a package called `bootstrapreport`, because after I deployed it got auto-renamed to `bootstrapreport1`. I don't have access to delete deployed packages to `jmslab` in [shinyapps](https://www.shinyapps.io/admin/#/applications/all), so if we want to re-deploy I think we need to delete both and redo it.

@MosesStewart there was an undeployed app called `bootstrapreport` which I've now archived and deleted. That should (?) clear the namespace for a new deployment. For now I haven't removed `bootstrapreport1` so that it remains available for testing.